contract/models/abstract_contract_line.py

```diff
@@ -166,11 +172,9 @@ def _compute_recurring_interval(self):
     def _compute_date_start(self):
         self._set_recurrence_field("date_start")
 
-    # pylint: disable=missing-return
-    @api.depends("contract_id.recurring_next_date", "contract_id.line_recurrence")
-    def _compute_recurring_next_date(self):
```

Comment:
Why are you removing this?

Reply:
Hi @pedrobaeza. Sorry for the delay for the response.

I removed this because it's redundant. Since we are inheriting from contract.recurrency.mixin.basic in the contract.line, it's redundant having the same on the abstract.contract.line.

Reply:
But this override was needed AFAIK.

Reply:
Mmmm i'm going to check it now, but one of the problems is that the contract never ends. This happens because this inherit. 
On a first step, I created the _compute_date_end function below. This, trigger a chain of computes (next_period_date_start compute that triggers the recurring_next_date compute). 
Then, I checked this function and I concluded that it was redundant and not necessary. 

I've been working with this changed since I created the MR and it's working very well.

If u want, I can let function as it was initially since is not having any impact on the code.